web/src/features/hfiCalculator/util.ts

```diff
@@ -19,7 +19,21 @@ export const getZoneFromAreaName = (areaName: string): string => {
   return areaName.slice(-3)
 }
 
+export const getDailiesForCSV = (
```

Comment:
I need some help understanding the flow here... we can chat.

Reply:
- `chain` -> wrap content in a lodash chain to allow chained transformation functions (https://lodash.com/docs/4.17.15#chain)
- `groupBy` -> groups the dailies by their station code so we have lists of dailies all the with same station code
(https://lodash.com/docs/4.17.15#groupBy)
- `map` -> The above `groupBy` leaves us with a `Dictionary` type which is hard to work with, so we map that into objects of { code, dailies }, we now have a list of objects containing dailies for each station code
- `value()` -> unwrap the chain
- `map()` -> for each object of station dailies, take only the selected number of prep days worth
- `flat()` -> flatten the list of dailies

Reply:
Oh sorry - I didn't mean that - bigger picture flow. Let's chat.